src/csvcubedmodels/rdf/dcat.py

```diff
@@ -119,6 +119,60 @@ def __init__(self, uri: str):
         Resource.__init__(self, uri)
         self.rdf_types.add(DCAT.Dataset)
 
+#The new class would be created here which also inherits the Resource class
+class Distribution(Resource):
+
+    isDistributionOf: Ann[
+        str, Triple(DCAT.isDistributionOf, PropertyStatus.recommended, URIRef)
+    ]
+    #these two will need changing probably
+    access_service: Ann[str, Triple(DCAT.accessService, PropertyStatus.mandatory, URIRef)]
+    access_url: Ann[str, Triple(DCAT.accessURL, PropertyStatus.mandatory, URIRef)]
+
+    byte_size: Ann[str, Triple(DCAT.byteSize, PropertyStatus.mandatory, Literal)]
```

Comment:
`byte_size` doesn't have the correct type of literal. Take another look at the DCATv2 standard.

Reply:
sorted